frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx

```diff
@@ -132,18 +132,17 @@ function DefinitionView({ group }: { group: TaxonomicFilterGroup }): JSX.Element
         return <></>
     }
 
+    const description: string | JSX.Element | undefined | null =
+        (definition && 'description' in definition && definition?.description) ||
+        (definition?.name &&
+            (keyMapping.element[definition.name]?.description || keyMapping.event[definition.name]?.description))
+
     const sharedComponents = (
         <>
-            {hasTaxonomyFeatures &&
-                definition &&
-                'description' in definition &&
-                (hasTaxonomyFeatures && definition.description ? (
```

Comment:
redundant `hasTaxonomyFeatures `

Reply:
yeah.. that's one of the reasons why I deleted these lines :D

Reply:
Oh man, I must have looked at the wrong section :D